src/gen_tw.erl

```diff
@@ -184,25 +184,24 @@ loop(LVT, _Events=[#event{lvt=GVT, payload=?GVT_UPDATE_PAYLOAD}|T], PastEvents,
     erlang:garbage_collect(),
     loop(LVT, T, NewPastEvents, Module, NewModStates);
 
-%% First event in queue occurs before LVT.  Rollback to handle the event.
+%% First event in queue is an antievent for an event in PastEvents.  In this
+%% case we roll back to a state occuring before the antievent, and resume
+%% processing.  Placing events in PastEvents back into Events ensures that
+%% the antievent and event will cancel each other out in the Events queue.
+loop(LVT, Events=[#event{lvt=ELVT, not_anti=false}|_], PastEvents, Module, ModStates)
+        when ELVT =< LVT ->
+
+    NewModStates = lists:dropwhile(fun({SLVT, _}) -> SLVT >= ELVT end, ModStates),
+    [{LastKnownLVT, _}|_] = NewModStates,
```

Comment:
We should never get an antievent at or before GVT.  The LVT we initialize at is assumed to be GVT.